sdk/inc/azure/iot/az_iot_pnp_client.h

```diff
@@ -582,18 +583,18 @@ AZ_NODISCARD az_result az_iot_pnp_client_twin_property_builder_end_component(
  * @endcode
  *
  * To send a status for a property belonging to a component, first call the
- * az_iot_pnp_client_twin_property_builder_begin_component() API to prefix the payload with the
+ * az_iot_pnp_client_property_builder_begin_component() API to prefix the payload with the
  * necessary identification. The API call flow would look like the following with the listed JSON
  * payload being generated.
  *
  * **With Component**
```

Comment:
Nit - it's unclear `Component` should be in caps here.  It may be per being in a heading, but component isn't capitilized in https://docs.microsoft.com/en-us/azure/iot-pnp/concepts-components

Reply:
that's fair. originally had it there more as a "title" but I'll lowercase to unify